.travis.yml

```diff
@@ -14,11 +14,14 @@ addons:
       - g++-4.8
 env:
   matrix:
-    - CXX=g++-4.8 TEST_SUITE=unit
+    - CXX=g++-4.8 TEST_SUITE=test
```

Comment:
@holgerd77 if you change here to `test` then you should remove `lint` task from `matrix.include`, otherwise maybe add extra task here?
```yml
env:
  matrix:
    CXX=g++-4.8 TEST_SUITE=integration
    CXX=g++-4.8 TEST_SUITE=unit
```

Reply:
@fanatid Thanks, that makes sense, changed to your second proposal. Can you do another review?